web_responsive/static/src/legacy/js/kanban_renderer_mobile.js

```diff
@@ -176,7 +176,7 @@ odoo.define("web_responsive.KanbanRendererMobile", function (require) {
          */
         _setState: function () {
             const res = this._super.apply(this, arguments);
-            this.columnOptions.recordsDraggable = !config.device.isMobile;
+            this.columnOptions.recordsDraggable = config.device.isMobile ? false : this.columnOptions.recordsDraggable;
```

Comment:
```python
            this.columnOptions.originRecordsDraggable =
                this.columnOptions.recordsDraggable;
            this.columnOptions.recordsDraggable =
                !config.device.isMobile && this.columnOptions.recordsDraggable;
```

Reply:
Great! That's what I means. Your solution is more comprehensive.
Thanks for your replay.

Reply:
If you will have troubles adapting your PR - let me know. I can help.